docs/simulation_operators_ground_sched.inc

```diff
@@ -0,0 +1,160 @@
+```{code-block} console
+
+usage: toast_ground_schedule [-h] [--site-name SITE_NAME]
```

Comment:
This file is checked in instead of being auto-generated?

Reply:
Good point, I will automate that during doc setup, when we already have the toast package loaded anyway.